src/Scene/NodeProcess.js

```diff
@@ -88,7 +88,7 @@ define('Scene/NodeProcess',['Scene/BoundingBox','Renderer/Camera','Core/Math/Mat
 
         var obb     = node.OBB();               
         
-        var l       = node.absoluteCenter.length();
+        var l       = 0 * node.absoluteCenter.length(); // TODO : is this really necessary? (note : bugs on plane mode if not 0)
```

Comment:
I didn't understand this part of the code. If I didn't put l = 0, my geometries wouldn't appear.